WordPress/src/main/java/org/xmlrpc/android/XMLRPCUtils.java

```diff
@@ -307,23 +305,17 @@ private static String verifyXmlrpcUrl(final String siteUrl, final String httpUse
     // whole process.
     private static String discoverSelfHostedXmlrpcUrl(String siteUrl, String httpUsername, String httpPassword) throws
             XMLRPCUtilsException {
-        // Array of Strings that contains the URLs we want to try
-        final List<String> urlsToTry = new ArrayList<>();
```

Comment:
Hehe, but we don't have "if not in array add it" anymore. :) 
I'm ok with it though.